website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -57,6 +57,51 @@ Use the Accordion when the toggle or the content is relatively complex.
 ![Accordion with complex content](/assets/components/accordion/usage-do-complex.png =770x*)
 !!!
 
+## Sizes
+
+The Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium` for most use cases, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, don't mix different sizes of the AccordionItem.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
```

Comment:
```suggestion
![Example of incorrectly combined Accordion sizes](/assets/components/accordion/accordion-size-Dont.png)
```

It's generally recommended to not include words like "image/picture of/here" in alt text. (Tagging @MelSumner in case she'd like to add anything.)

(Also btw, I would recommend using all lowercase letters in file names, ie no caps.)